airflow/cli/cli_parser.py

```diff
@@ -884,7 +893,9 @@ class GroupCommand(NamedTuple):
             "Run subsections of a DAG for a specified date range. If reset_dag_run option is used, "
             "backfill will first prompt users whether airflow should clear all the previous dag_run and "
             "task_instances within the backfill date range. If rerun_failed_tasks is used, backfill "
-            "will auto re-run the previous failed task instances  within the backfill date range"
+            "will auto re-run the previous failed task instances  within the backfill date range. If "
+            "rerun_succeeded_tasks is used, backfill will auto re-run the previous failed task instances "
```

Comment:
```suggestion
            "rerun_succeeded_tasks is used, backfill will auto re-run the previous succeeded task instances "
```
I think that's what you want to do, right?

Reply:
yes